src/modules/sub-items/components/bulk-delete/bulk.delete.btn.js

```diff
@@ -0,0 +1,163 @@
+                    {},
+                    'sub_items'
+                );
+
+                window.eZ.helpers.notification.showErrorNotification(message);
+            }
+        });
+    }
+
+    togglePopup(show) {
+        this.setState(() => ({
+            isPopupVisible: show,
+        }));
+    }
+
+    closePopup() {
+        this.togglePopup(false);
+    }
+
+    onPopupConfirm() {
+        this.closePopup();
+        this.bulkDelete();
+    }
+
+    renderConfirmationPopupFooter() {
+        const cancelLabel = Translator.trans(/*@Desc("Cancel")*/ 'bulk_delete.popup.cancel', {}, 'sub_items');
+        const confirmLabel = Translator.trans(/*@Desc("Send to trash")*/ 'bulk_delete.popup.confirm', {}, 'sub_items');
+
+        return (
+            <Fragment>
+                <button onClick={this.closePopup} type="button" className="btn btn-secondary btn--no" data-dismiss="modal">
+                    {cancelLabel}
+                </button>
+                <button
+                    onClick={this.onPopupConfirm}
+                    className="btn btn-danger font-weight-bold btn--trigger"
+                    data-click="#version-remove-version_remove_draft_remove"
+                >
+                    {confirmLabel}
+                </button>
+            </Fragment>
+        );
+    }
+
+    renderConfirmationPopup() {
+        const { isPopupVisible } = this.state;
+
+        if (!isPopupVisible) {
+            return null;
+        }
+
+        const confirmationMessage = Translator.trans(
+            /*@Desc("Are you sure you want to send the selected content item(s) to the trash?")*/ 'bulk_delete.popup.message',
+            {},
+            'sub_items'
+        );
+
+        return ReactDOM.createPortal(
+            <Popup
+                onClose={this.closePopup}
+                isVisible={isPopupVisible}
+                isLoading={false}
+                size="medium"
+                footerChildren={this.renderConfirmationPopupFooter()}
+                plain
+            >
+                <div className="c-bulk-delete-btn__modal-body">{confirmationMessage}</div>
+            </Popup>,
+            this.modalContainer
+        );
+    }
+
+    render() {
+        const { selectedItems } = this.props;
+        const label = Translator.trans(/*@Desc("Delete selected items")*/ 'trash_btn.label', {}, 'sub_items');
+        const disabled = !selectedItems.length;
+        const baseClassName = 'm-sub-items__btn';
+        let className = `${baseClassName} ${baseClassName}--trash`;
+
+        className = disabled ? `${className} ${baseClassName}--disabled` : className;
+
+        return (
+            <Fragment>
+                <div className={className} title={label} onClick={this.onBtnClick}>
+                    <svg className="ez-icon">
```

Comment:
You should use `ez-icon--medium` and remove CSS properties related to the icon dimensions, as well.